src/contract/assembled_transaction.ts

```diff
@@ -384,39 +382,49 @@ export class AssembledTransaction<T> {
    * Serialize the AssembledTransaction to a base64-encoded XDR string.
    */
   toXDR(): string {
-    if(!this.built) throw new Error(
+    if (!this.built)
+      throw new Error(
         "Transaction has not yet been simulated; " +
-        "call `AssembledTransaction.simulate` first.",
+          "call `AssembledTransaction.simulate` first."
       );
-    return this.built?.toEnvelope().toXDR('base64');
+    return this.built?.toEnvelope().toXDR("base64");
   }
 
   /**
    * Deserialize the AssembledTransaction from a base64-encoded XDR string.
    */
   static fromXDR<T>(
-    options: Omit<AssembledTransactionOptions<T>, "args" | "method" | "parseResultXdr">,
+    options: Omit<
+      AssembledTransactionOptions<T>,
+      "args" | "method" | "parseResultXdr"
+    >,
     encodedXDR: string,
     spec: Spec
   ): AssembledTransaction<T> {
     const envelope = xdr.TransactionEnvelope.fromXDR(encodedXDR, "base64");
-    const built = TransactionBuilder.fromXDR(envelope, options.networkPassphrase) as Tx;
+    const built = TransactionBuilder.fromXDR(
+      envelope,
+      options.networkPassphrase
+    ) as Tx;
     const operation = built.operations[0] as Operation.InvokeHostFunction;
-    if (!operation?.func?.value || typeof operation.func.value !== 'function') {
-      throw new Error("Could not extract the method from the transaction envelope.");
+    if (!operation?.func?.value || typeof operation.func.value !== "function") {
+      throw new Error(
+        "Could not extract the method from the transaction envelope."
+      );
     }
     const invokeContractArgs = operation.func.value() as xdr.InvokeContractArgs;
     if (!invokeContractArgs?.functionName) {
-      throw new Error("Could not extract the method name from the transaction envelope.");
+      throw new Error(
+        "Could not extract the method name from the transaction envelope."
+      );
```

Comment:
There are clearly still many changes in this PR exclusively related to formatting. It makes it very hard to safely check the actual changes relevant to the issue at hand. Please check out the repository, ONLY add your changes, then commit and push those. Do not run any sort of formatter, automatic or otherwise. We **cannot** review or merge if there are many _many_ lines which just change the code formatting.